doc/advanced-testing-mini-course/src/ATMC/Lec5/ViewstampReplication/Machine.hs

```diff
@@ -91,7 +84,6 @@ current view-number, m is the message it received
 from the client, n is the op-number it assigned to
 the request, and k is the commit-number.
   -}
-    else do
       tODO
```

Comment:
I can't make it a suggestion but this `tODO` should not be indented (it was because of the `if` before).